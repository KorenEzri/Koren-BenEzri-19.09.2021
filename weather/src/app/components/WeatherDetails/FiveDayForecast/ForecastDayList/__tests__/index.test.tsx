import * as React from 'react';
import { render } from '@testing-library/react';

import { ForecastDayList } from '..';


describe('<ForecastDayList  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(<ForecastDayList />);
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
