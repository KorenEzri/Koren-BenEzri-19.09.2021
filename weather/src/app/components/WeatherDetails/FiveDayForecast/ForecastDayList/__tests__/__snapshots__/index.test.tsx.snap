// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ForecastDayList  /> should match snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  list-style-type: none;
  max-width: 1100px;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
}

<ul
  class="c0"
/>
`;
